### DISCUSSION TITLE
Support for Thread-bases version of Qingping Temp & RH Monitor?

### DISCUSSION THREAD

**Discussion by tomillr** [pvvx/ATC_MiThermometer#517] [Ideas]
Qingping makes a version of the Qingping Temp & RH Monitor that uses the new Thread-protocol. This version is listed as the T-version (CGG1T) instead of the M-version (CGG1M).

https://www.qingping.co/temp-rh-monitor/overview

Would it be possible to provide a custom firmware for the T-version as well to improve the battery life to the device?

**Comment by tomillr** [comment]
Anyone?

**Reply by m4dm4xi** [reply]
Did you find any solution @tomillr ?

**Reply by pvvx** [reply]
Nobody even deigned to indicate the SoC type in this device.

**Comment by Session-78** [comment]
Would also be interested but have no coding experience..
